scipy/stats/tests/test_continuous.py

```diff
@@ -894,6 +894,25 @@ def _moment_raw_formula(self, order, **params):
         with pytest.raises(ValueError, match=message):
             X.validation_policy = "invalid"
 
+    def test_shapes(self):
+        X = stats.Normal(mu=1, sigma=2)
+        Y = stats.Normal(mu=[2], sigma=3)
+
+        # Check that attributes are available as expected
+        assert X.mu == 1
+        assert X.sigma == 2
+        assert Y.mu[0] == 2
+        assert Y.sigma[0] == 3
+
+        # Trying to set an attribute raises
+        message = "property of 'Normal' object has no setter"
```

Comment:
```suggestion
        message = "...'Normal' object has no setter"
```

Reply:
Any thoughts on adding a custom `fset` in the `property` that raises with a more informative message?

Reply:
I think we eventually want to use `fset` to allow changing parameters. Until then, I'm happy with the default message, personally. 

There are two hold-ups:
- When parameters are changed, an `_update_parameters` method needs to be called to precompute quantities used by many methods. For instance, in the uniform distribution, you'd want in to compute the width of the support `b - a` once and use it in practically every method. But if you're going to change both `a` and `b` , you don't want `_update_parameters` to be called twice, so we talked about updating lazily, and only running `_update_parameters` when needed.
- it's unclear what to do with transformed distributions. Right now, if you have say `Y = exp(X)`, changing `X` will not change `Y`. As we have it right now, changing the components of a mixture *would* change the mixture. But what we want needs to be considered carefully.

Reply:
OK, cool. Sounds good to me.

Reply:
Do you plan to commit the suggestion above? I don't really understand it. I think `pytest.raises` uses `re.search` under the hood, so it should be sufficient to just have `"'Normal' object has no setter"`? Doesn't `...` just mean it matches three arbitrary characters?

Reply:
I'll try that locally before merging. The point is that different versions of Python seem to emit different messages, so I'm just trying to write something reasonably specific but that will match both.

Reply:
I think you're right, but there are many cases in stats tests in which `...` is included - maybe just to indicate to the reviewer that this is not the entire message.

Reply:
Wow, there are actually at least three different possible mentions, and one wouldn't match ("can't set attribute"). I'm going to leave the message empty and comment about why.